### DISCUSSION TITLE
Generate HTML Tag after hit Tab button

### DISCUSSION THREAD

**Discussion by hafidluqman50** [zed-industries/zed#8716] [Feature Requests]
Hey, can we have some simple generate tags HTML by just using tab? 

i used to be using Sublime Text 4 when that text editor also have some package called emmet and alignment. 

Emmet allowing generate HTML tags with class just by define the class and hit the tab button, and for the alignment it can useful for tidy up the array for having same tabs and line to make it clean. Just a little wish from me btw. 

i hope next update we got some feature similiar like this. 🙏

**Comment by Moshyfawn** [comment]
This is already a feature request in #4992. Be sure to +👍 the issue to show your interest.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
